src/foam/lib/csv/CSVNormalStringParser.java

```diff
@@ -8,6 +8,15 @@
 
 import foam.lib.parse.*;
 
+/**
+ * The class parse the CSV normal string base on the CSV string rule
+ * The string can not contain '"', '\r', '\n' and ','
+ * eg:
+ *  abc123    : legal
+ *  ab"e123   : illegal
+ *  ab\ne123  : illegal
+ *  ab\re123  : illegal
+ */
```

Comment:
So NormalStringParser is a strict subset of CSVEscapeStringParser.

What is the value is keeping NormalStringParser?  Do we ever need to be this strict when parsin?